projects/nyancat/description.yaml

```diff
@@ -0,0 +1,39 @@
+name: NYAnCAT
+authors:
+  - dev.bo: https://github.com/devdotbo
+description: |
+  NYAnCAT: The cutest and sharpest AI personal banking cat that brings you
+  low-risk returns
+
+  NYAnCAT is a World APP Mini App that can serve four roles: AI banking advisor,
+  fund manager, AI news source, and governance tool.
+
+  -As a personal banking AI advisor,  it can use ML to search for low-risk
+  investment opportunities on CEXs.
+
+  -As a fund manager, it can execute strategies and collect revenues.
+
+  -As an AI news source, it can collect the latest market trends and present
+  them to you in a few words.
+
+  -As a governance tool, you can use it to vote for your favourite strategies.  
```

Comment:
```suggestion
  - As a personal banking AI advisor,  it can use ML to search for low-risk
    investment opportunities on CEXs.

  - As a fund manager, it can execute strategies and collect revenues.

  - As an AI news source, it can collect the latest market trends and present
    them to you in a few words.

  - As a governance tool, you can use it to vote for your favourite strategies.  
```